### ISSUE TITLE
Tableau does not recognize the version of the web data connector API.

### ISSUE THREAD

**Issue by xicubed** [tableau/webdataconnector#119]
Started going through the Examples in the dev branch (which use https://connectors.tableau.com/libs/tableauwdc-2.1.1.js) with a connector from within Tableau 10.0.2 (10000.16.1004.1720) 64-bit Mac and get the error:

An error occurred while communicating with the data Source. Tableau does not recognize the version of the web data connector API. Show Details: The version of Tableau that you are using cannot use the web data connector that you are trying to access. The connector requires at least version '2.1.1' of the web data connector API.

If I substitute `https://connectors.tableau.com/libs/tableauwdc-2.1.1.js` in the earthquakeUSGS.html with `https://connectors.tableau.com/libs/tableauwdc-2.0.beta.js` it works fine using a Web Data Connector from within Tableau.
![screen shot 2016-10-21 at 11 02 55 am](https://cloud.githubusercontent.com/assets/19937/19606289/87ff07f0-977e-11e6-8157-6dd8438e783f.png)

In short it's erring out while at the same time telling me I need to use 2.1.1 which I am using...

**Comment by lbrendanl** [comment]
@xicubed Thanks for bringing this to our attention, the docs change is an error, apologies for the hassle! Please use 2.0.latest (2.0.9).   We will make version compatibility clearer in the docs in future.  

@jdomingu can you roll back the doc change for the version number?

**Comment by jdomingu** [comment]
@lbrendanl Should be done now :)

**Comment by xicubed** [comment]
Thanks!
